queue_services/payment-reconciliations/q_cli.py

```diff
@@ -85,7 +84,7 @@ def subscription_options():
 
         payload = {
             'specversion': '1.x-wip',
-            'type': 'bc.registry.payment.casSettlementUploaded',
+            'type': f'{message_type}',
             'source': 'https://api.business.bcregistry.gov.bc.ca/v1/business/BC1234567/filing/12345678',
```

Comment:
Is it better to use config or read input instead of hard code in the program?

Reply:
Sorry, didnt get this. q_cli.py was having wrong values earlier

Reply:
I mean the "source".